resources/i18n/ko.json

```diff
@@ -13,7 +13,7 @@
             "Configurations": "환경 설정",
             "Maintenance": "관리",
             "Information": "정보",
-            "About": "대하여",
+            "AboutBackend.AI": "Backend.AI에 대하여",
```

Comment:
In translation part, dot (.) is used to divide namespaces. How about removing dot from AboutBackend.AI? (to AboutBackendAI)